Unit-01/02-flask-routing/app.py

```diff
@@ -0,0 +1,34 @@
+from flask import Flask
+
+
+app = Flask(__name__)
+
+@app.route('/add/<int:num1>/<int:num2>')
+def add(num1, num2):
+	return "{}".format(num1 + num2)
+
+@app.route('/subtract/<int:num1>/<int:num2>')
+def subtract(num1, num2):
+	return "{}".format(num1 - num2)
+
+@app.route('/multiply/<int:num1>/<int:num2>')
+def multiply(num1, num2):
+	return "{}".format(num1 * num2)
+
+@app.route('/divide/<int:num1>/<int:num2>')
+def divide(num1, num2):
+	return "{}".format(int(num1 / num2))
+
+@app.route('/math/<operation>/<int:num1>/<int:num2>')
+def math(operation, num1, num2):
+	if operation == 'add':
+		return "{}".format(num1 + num2)
+	if operation == 'subtract':
```

Comment:
Great Job!

Two things:

1. I would definitely make these `elif`s since they are mutually-exclusive.

1. However, another way to do this is using a dictionary, just like you can in JavaScript using objects.

```python
def add(a, b):
    return a + b
def subtract(a, b):
    return a - b
def multiply(a, b):
    return a * b
def divide(a / b):
    return a / b

operations = {
    "add": add,
    "subtract": subtract,
    "multiply": multiply,
    "divide": divide
}

# @route handler
def math(operation, num1, num2):
    if operation in operations:
        return f"{operations[operation](num1, num2)}"
```

This is a more declarative style, where you can declare your functions and data structures at the time and simplify the business logic part (i.e. the routing, in this case)